portableLib/OAT/customEvent.js

```diff
@@ -0,0 +1,110 @@
+/**
```

Comment:
Why do you need to have a copy of the `portableLib`.
FYI  `portableLib` are libraries proposed to PCI and PIC implementer to be reused in their custom interaction/info control. 
If this is only for the tests, we will need to find a way to not have them part of this repository.

Reply:
It was necessary for `portableElementRegistry`, but it is not in this repository anymore. So I removed them.